items.json

```diff
@@ -10269,18 +10269,18 @@
         "missileSpeed": 0,
         "stackAmount": 0,
         "length": 100,
-        "balance": 0.51,
-        "handling": 89,
+        "balance": 0.0,
+        "handling": 58,
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon"
         ],
         "thrustDamage": 0,
         "thrustDamageType": "Undefined",
-        "thrustSpeed": 90,
+        "thrustSpeed": 48,
         "swingDamage": 38,
         "swingDamageType": "Blunt",
-        "swingSpeed": 85
+        "swingSpeed": 25

```

Comment:
Are you sure you want a weapon that slow? There is a huge chance it crashes the game...

Reply:
WHOOPS I TOTALLY FAT FINGERS THE 0 I MEANT FOR IT TO BE 1 NOT 10